specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionsConfirmConsentCode.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "connectionName": "testManagedApi",
+    "api-version": "2016-06-01",
+    "confirmConsentCode": {
+      "code": "testCode"
```

Comment:
objectId and tenantId could also be provided by users (I am not sure what the goal of these example is; if it is to provide just a sample mainline request, then what you have is good; if it is to provide an exhaustive sample covering all properties, then let's add tenant id and object id)

Reply:
My understanding is that it is to provide samples for AutoRest to use as a validation. So we should definitely use the most complete requests and responses. Some of these I extracted the bodies from reading the unit tests for them so they might be missing things.